views/time-series.svg

```diff
@@ -89,9 +93,19 @@
     
 <g class="polls" transform="translate({{margin.left}},{{margin.top}})">
 {% for poll in data %}
-    <circle cx="{{poll.x}}" cy="{{poll.y.remain}}" r="{{pollStyle.radius}}" fill="{{pollStyle.remainFill}}" fill-opacity="{{pollStyle.fillOpacity}}"></circle>
-    <circle cx="{{poll.x}}" cy="{{poll.y.leave}}" r="{{pollStyle.radius}}" fill="{{pollStyle.leaveFill}}" fill-opacity="{{pollStyle.fillOpacity}}"></circle>
-    <circle cx="{{poll.x}}" cy="{{poll.y.undecided}}" r="{{pollStyle.radius}}" fill="{{pollStyle.undecidedFill}}" fill-opacity="{{pollStyle.fillOpacity}}"></circle>
+    {% if poll.data.method == 'Telephone Interviews' %}
+        <circle cx="{{poll.x}}" cy="{{poll.y.remain}}" r="{{pollStyle.radius}}" fill="{{pollStyle.remainFill}}" fill-opacity="{{pollStyle.fillOpacity}}" stroke="#74736c"></circle>
```

Comment:
Couldn't all this just be a condition around the stroke attribute. The changed order of the layers could be for all?